cmd/puppeth/ssh.go

```diff
@@ -129,15 +129,20 @@ func dial(server string, pubkey []byte) (*sshClient, error) {
 			fmt.Printf("SSH key fingerprint is %s [MD5]\n", ssh.FingerprintLegacyMD5(key))
 			fmt.Printf("Are you sure you want to continue connecting (yes/no)? ")
 
-			text, err := bufio.NewReader(os.Stdin).ReadString('\n')
-			switch {
-			case err != nil:
-				return err
-			case strings.TrimSpace(text) == "yes":
-				pubkey = key.Marshal()
-				return nil
-			default:
-				return fmt.Errorf("unknown auth choice: %v", text)
+			for {
+				text, err := bufio.NewReader(os.Stdin).ReadString('\n')
+				switch {
+				case err != nil:
+					return err
+				case strings.TrimSpace(text) == "yes":
+					pubkey = key.Marshal()
+					return nil
+				case strings.TrimSpace(text) == "y", strings.TrimSpace(text) == "n":
+					fmt.Println("please answer 'yes' or 'no'")
+					continue
+				default:
+					return fmt.Errorf("unknown auth choice: %v", text)
```

Comment:
Maybe we could make the `fmt.Println("please answer 'yes' or 'no'")` as the default branch. Also please Capitalize `please`.

Reply:
Ha, we still need `no` to fail, so can't make it default.

Reply:
Ok, let's handle `yes`, `no`, and default anything else into the loop